test/simple/test-tls-securepair-fiftharg.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var common = require('../common');
```

Comment:
I have said in another comment, that there is no requirements to use ES6 features. If io.js claim to suppoer ES6 completely, I will fix.

If you strongly require to fix that, I will fix anyway.